src/validation.cpp

```diff
@@ -5681,6 +5681,10 @@ util::Result<void> ChainstateManager::ActivateSnapshot(
             return util::Error{strprintf(_("The base block header (%s) is part of an invalid chain."), base_blockhash.ToString())};
         }
 
+        if (!m_best_header || m_best_header->GetAncestor(base_blockheight) != snapshot_start_block) {
+            return util::Error{_("A forked headers-chain with more work than the chain with the base block header exists. Please proceed to sync without AssumeUtxo.")};
```

Comment:
nit: seems the "snapshot" added before is back out, still not a block for me though

Reply:
ah, sorry, am on a different computer now and forgot to update the branch... Fixed now.